File: ch1/1-4.go
package main

import "fmt"

func main() {
	var fahr, celsius float64
	var upper, step int

	upper = 300
	step = 20

	fmt.Printf("Celsius\t\tFahrenheit\n")
	for celsius <= float64(upper) {
		fahr = (((celsius * 9.0) / 5) + 32)
		fmt.Printf("%3.0f\t\t%6.1f\n", celsius, fahr)
		celsius = celsius + float64(step)
	}
}

File: kr/ch1/1-4.c
#include <stdio.h>

int main(void)
{
    float fahr, celsius;
    int lower, upper, step;

    lower   = 0;
    upper   = 300;
    step    = 20;
    celsius = lower;

    printf("Celsius\t\tFahrenheit\n");
    while (celsius <= upper) {
        fahr = (((celsius * 9.0) / 5) + 32);
        printf("%3.0f\t\t%6.1f\n", celsius, fahr);
        celsius = celsius + step;
    }
}
